### ISSUE TITLE
Display exception handlers in the GUI

### ISSUE THREAD

**Issue by torusrxxx** [x64dbg/x64dbg#837]
Exception handlers can be displayed in the log. We can also display them in the GUI.

**Comment by mrexodia** [comment]
Yes I agree. My idea was to simply put them in the references view 😀 but
we could also use the SEH view.

However they need a bit more work because currently you need symbol files
to get them...
